app_flutter/lib/status_grid.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'result_box.dart';
+
+class StatusGrid extends StatelessWidget {
+  static const int taskCount = 80; // rough estimate based on existing dashboard
```

Comment:
Generally good practice to add a const constructor that takes a key:

`const StatusGrid({Key key}) : super(key: key);`

Reply:
Added!